### DISCUSSION TITLE
I get a error for this?

### DISCUSSION THREAD

**Discussion by DinhHuy2010** [ThioJoe/YT-Spammer-Purge#223] [Q&A] [answered]
Could anyone fix this error? Get the error while scanning spammers by sensitive mode or non-ascii mode.
Error: 
The deletion functionality was not enabled. Cannot delete or report comments.
Possible Cause: You're scanning someone elses video with a non-supported filter mode.
![Screenshot 2021-12-27 123841](https://user-images.githubusercontent.com/95196459/147438235-42c73869-379c-4164-8161-ab7fa1bbd0df.png)

**Comment by kiatchun** [comment]
Is it your own channel or someone else?

**Reply by DinhHuy2010** [reply]
someone else

**Comment by kiatchun** [comment, marked as answer]
When scanning someone else's vid, you can only report when using normal mode (option 1). 

<img width="711" alt="image" src="https://user-images.githubusercontent.com/93459510/147445566-d7dfb38a-467d-4ba7-8088-386caf698b85.png">

**Reply by DinhHuy2010** [reply]
ok
